media-libs/sdl-net/sdl-net-1.2.8-r2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multilib-minimal toolchain-funcs
+
+MY_P="${P/sdl-/SDL_}"
+DESCRIPTION="Simple Direct Media Layer Network Support Library"
+HOMEPAGE="https://www.libsdl.org/projects/SDL_net/index.html"
+SRC_URI="https://www.libsdl.org/projects/SDL_net/release/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-solaris"
+IUSE="static-libs"
+
+RDEPEND="media-libs/libsdl[${MULTILIB_USEDEP}]"
+DEPEND="${RDEPEND}"
+
+multilib_src_configure() {
+	AS="$(tc-getAS)" ECONF_SOURCE="${S}" econf \
+		--disable-gui \
+		$(use_enable static-libs static)
+}
+
+multilib_src_install_all() {
+	dodoc CHANGES README
+	if ! use static-libs ; then
+		find "${D}" -type f -name '*.la' -exec rm {} + || die
+	fi
```

Comment:
Make deletion unconditional but use the find statement from the policy guide:
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0303